S23/pmirukula/Conversion.java

```diff
@@ -0,0 +1,44 @@
+import java.util.*;
+import java.io.BufferedReader;
+import java.io.InputStreamReader;
+import java.io.*;
+import java.util.Locale;
+
+public class Conversion {
+	public void conversionlogic() 
+	{
+	}
+
+	public static void main(String[] args) throws IOException {
+		BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
+		System.out.println("Please enter the word");
+		String str = br.readLine();
+		System.out.println("Please enter the language you would like to translate the word");
+		String str_lang = br.readLine();
+		PrintWriter out = new PrintWriter("oceans.txt"); 
+		String output = str.toLowerCase(new Locale(str_lang));
+		if(str_lang == " tr " || str_lang == "az")
+		{
+			if(str.contains("I")) 
+			{
+				str = str.replaceAll("I", " \0u131");
+				output = str.toLowerCase(new Locale(str_lang));
+			}
+		}
+		else if(str_lang.equals("ga-IE")|| str_lang.equals("ga") )

```

Comment:
// maybe it could be better to handle the string in a way there we can separate two part of the language. It could be awesome if we can split the string with "-" sign so we can just compare it with "ga".